cdk/lib/FargateServiceStack.ts

```diff
@@ -58,7 +58,8 @@ export class FargateServiceStack extends Stack {
       taskDefinition: taskDef,
       environment: {
         SQS_URL: queue.queueUrl,
-        DDB_TABLE: ddbTable.tableName
+        DDB_TABLE: ddbTable.tableName,
+        AWS_PROFILE: "ls-sandbox"
```

Comment:
did you add "ls-sandbox" profile intentionally?

Reply:
good catch! that needs to go since it was machine specific, but it has no impact anymore anyway